doc/contributing/releases.md

```diff
@@ -605,10 +605,14 @@ the build before moving forward. Use the following list as a baseline:
 * Run the test suite against the built binaries (optional)
 
 ```console
-./tools/test.py --shell ~/Downloads/node-v18.5.0-linux-x64/bin/node
+ # some tests are relying on the filesystem and the binary name
+ cp ~/Downloads/node-v18.5.0-linux-x64/bin/node ./out/Release/node
+./tools/test.py --shell ./out/Release/node
```

Comment:
If you copy the binary you want to test to `./out/Release/node` there's no need to pass `--shell` to the test runner as that is the default path to the binary to test.